packages/blocks/src/blocks/core-image.tsx

```diff
@@ -55,7 +55,7 @@ interface MediaSize {
 	name: string;
 	sourceUrl: string;
 }
-
+//@ts-ignore
```

Comment:
Anywhere we `@ts-ignore` we should leave a reason  e.g. 
```suggestion
// @ts-ignore: Unused until lightbox support is fixed.
```